content/source/docs/enterprise/sentinel/import/tfstate.html.md

```diff
@@ -10,103 +10,526 @@ description: |-
 
 The tfstate import provides access to the Terraform state.
 
-### tfstate.module_paths
+This is the Terraform state as it exists at plan time. Depending on the state of
+a Terraform workspace, the data in this import may be incomplete (such as when a
+resource has not been created yet).
+
+Depending on what you are looking for, the [`tfconfig`][import-tfconfig] and
+[`tfplan`][import-tfplan] imports can assist with providing a more complete view
+of the state and configuration of the workspace as it exists currently, or in
+the future as well. `tfplan` also has aliases to both the `tfconfig` and
+`tfplan` imports to help simplify policy.
+
+[import-tfconfig]: /docs/enterprise/sentinel/import/tfconfig.html
+[import-tfplan]: /docs/enterprise/sentinel/import/tfplan.html
+
+-> To reiterate, this import works with the Terraform state that _currently
```

Comment:
Per Nick's recommendations, this should have a **bold** styled header saying what it is (`Note:` I think, in this case)

Reply:
Just talked about this offline - there's a number of info boxes where this would need to be updated if this is a convention we want to stick to. Leaving this here for now, pending @nfagerlund's comments.

Reply:
Ironically, a re-write put context into the info box that more warranted a **NOTE** heading. :stuck_out_tongue: